Cargo.toml

```diff
@@ -24,7 +24,7 @@ exclude = [
 members = [
 	"node",
 	"pallets/*",
-	"pop-api/integration-tests",
+	#"pop-api/integration-tests",
```

Comment:
reason for this?

Reply:
Currently breaking compilation and is out of scope for this PR